pkg/mobile/types.go

```diff
@@ -33,6 +34,18 @@ type DeviceMetric struct {
 	PlatformVersion string `json:"platformVersion"`
 }
 
+type SecurityMetrics struct {
+	EmulatorCheck      *SecurityMetric `json:"org.aerogear.mobile.security.checks.EmulatorCheck,omitempty,string"`
+	DeveloperModeCheck *SecurityMetric `json:"org.aerogear.mobile.security.checks.DeveloperModeCheck,omitempty,string"`
+	DebuggerCheck      *SecurityMetric `json:"org.aerogear.mobile.security.checks.DebuggerCheck,omitempty,string"`
+	RootedCheck        *SecurityMetric `json:"org.aerogear.mobile.security.checks.RootedCheck,omitempty,string"`
+	ScreenLockCheck    *SecurityMetric `json:"org.aerogear.mobile.security.checks.ScreenLockCheck,omitempty,string"`
```

Comment:
Will this make new check implementations require a new version of the api?

Reply:
Absolutely. Unfortunately that's how it's been implemented on the client side. I'm open to suggestions on how we can improve the client payload.

Reply:
Hrm, there is no way to the accept whatever string on the key, so we don't need to add a new line for every new check?

Reply:
@danielpassos The only way to do that is to have absolutely no validation on the entire `security` field. But then it would be possible to put anything in there.

Reply:
@darahayes do you know if there's a way to traverse the parsed json without having fields in a struct explicitly to receive the parsed results?
So we could do something like:
```
for key in securityMetric:
  assert securityMetricJson.get(key).hasBoolean(passed)
end
```

I guess for go this could be hard with all the explicit memory allocations...

Also this could be overkill for this use case, but might come in handy for future kinds of metrics.